examples/kubernetes/simple/k8s-manifests.yaml

```diff
@@ -8,6 +8,7 @@ stringData:
   POSTGRES_PASSWORD: "superSecretPassword"
   POSTGRES_USER: "optuna"
   POSTGRES_DB: "optunaDatabase"
+  POSTGRES_MLFLOW_DB: "mlflowDatabase"
```

Comment:
For the `simple/sklearn_distributed.py`, it might not need to add the database for `mlflow`. In my environment, I cannot execute commands in `simple/README.md` with the following errors. 
```
(venv) HideakinoMacBook-puro:simple mamu$ eval $(minikube docker-env)

(venv) HideakinoMacBook-puro:simple mamu$ docker image build -t optuna-kubernetes:example .
Sending build context to Docker daemon  9.728kB
Step 1/5 : FROM python:3.5-slim-buster
...
Successfully built 63fc91271bd6
Successfully tagged optuna-kubernetes:example

(venv) HideakinoMacBook-puro:simple mamu$ kubectl apply -f k8s-manifests.yaml
secret/postgres-secrets created
service/postgres created
statefulset.apps/postgres created
service/mlflow-server created
statefulset.apps/mlflow-server created
job.batch/worker created

(venv) HideakinoMacBook-puro:simple mamu$ kubectl logs worker-bq8xx
Error from server (BadRequest): container "worker" in pod "worker-bq8xx" is waiting to start: PodInitializing

(venv) HideakinoMacBook-puro:simple mamu$ kubectl get pod
NAME           READY   STATUS                  RESTARTS   AGE
postgres-0     1/1     Running                 0          2m24s
worker-bq8xx   0/1     Init:ErrImagePull       0          2m24s
worker-dl6c9   0/1     Init:ImagePullBackOff   0          2m24s
worker-hcrx5   0/1     Init:ErrImagePull       0          2m24s
worker-srm6x   0/1     Init:ImagePullBackOff   0          2m24s
worker-tmq9c   0/1     Init:ErrImagePull       0          2m24s
```

I'm not sure why the above `Init:ErrImagePull` and `Init:ImagePullBackOff` occur, but if I revert the changes on this file (`simple/k8s-manifest.yaml`), then the commands in `simple/README.md` succeed as follows.

```
...
(venv) HideakinoMacBook-puro:simple mamu$ kubectl logs worker-8wbpm
[I 2020-07-07 08:37:19,607] Using an existing study with name 'kubernetes' instead of creating a new one.
[I 2020-07-07 08:37:22,450] Finished trial#1 with value: 0.9533333333333333 with parameters: {'rf_max_depth': 25.03447162508479, 'classifier': 'RandomForest'}. Best is trial#2 with value: 0.96.
[I 2020-07-07 08:37:22,846] Finished trial#8 with value: 0.96 with parameters: {'rf_max_depth': 19.274788757800707, 'classifier': 'RandomForest'}. Best is trial#2 with value: 0.96.
[I 2020-07-07 08:37:23,046] Finished trial#11 with value: 0.9666666666666667 with parameters: {'rf_max_depth': 5.3908694340333705}. Best is trial#11 with value: 0.9666666666666667.
[I 2020-07-07 08:37:23,296] Finished trial#14 with value: 0.9533333333333333 with parameters: {'rf_max_depth': 11.66379043901938, 'classifier': 'RandomForest'}. Best is trial#11 with value: 0.9666666666666667.
[I 2020-07-07 08:37:23,468] Finished trial#16 with value: 0.96 with parameters: {'svc_c': 69779419.14216216, 'classifier': 'SVC'}. Best is trial#11 with value: 0.9666666666666667.
[I 2020-07-07 08:37:23,609] Finished trial#19 with value: 0.96 with parameters: {'svc_c': 469934.4329153692, 'classifier': 'SVC'}. Best is trial#11 with value: 0.9666666666666667.
[I 2020-07-07 08:37:23,806] Finished trial#20 with value: 0.9666666666666667 with parameters: {'rf_max_depth': 9.588754422871336, 'classifier': 'RandomForest'}. Best is trial#11 with value: 0.9666666666666667.
[I 2020-07-07 08:37:24,069] Finished trial#23 with value: 0.9466666666666667 with parameters: {'rf_max_depth': 9.708212392196772, 'classifier': 'RandomForest'}. Best is trial#11 with value: 0.9666666666666667.
[I 2020-07-07 08:37:24,312] Finished trial#26 with value: 0.9733333333333333 with parameters: {'svc_c': 2.42562154979395, 'classifier': 'SVC'}. Best is trial#26 with value: 0.9733333333333333.
[I 2020-07-07 08:37:24,576] Finished trial#28 with value: 0.9733333333333333 with parameters: {'svc_c': 2.514119235323489, 'classifier': 'SVC'}. Best is trial#26 with value: 0.9733333333333333.
[I 2020-07-07 08:37:24,801] Finished trial#31 with value: 0.9733333333333333 with parameters: {'svc_c': 3.2416253526631817, 'classifier': 'SVC'}. Best is trial#26 with value: 0.9733333333333333.
[I 2020-07-07 08:37:24,946] Finished trial#34 with value: 0.96 with parameters: {'svc_c': 146.99118603968049, 'classifier': 'SVC'}. Best is trial#32 with value: 0.98.
[I 2020-07-07 08:37:25,127] Finished trial#37 with value: 0.32 with parameters: {'svc_c': 0.00987266438110019, 'classifier': 'SVC'}. Best is trial#32 with value: 0.98.
[I 2020-07-07 08:37:25,341] Finished trial#39 with value: 0.96 with parameters: {'svc_c': 610.7051090756223, 'classifier': 'SVC'}. Best is trial#32 with value: 0.98.
[I 2020-07-07 08:37:25,550] Finished trial#42 with value: 0.9666666666666667 with parameters: {'svc_c': 0.4013340614641794, 'classifier': 'SVC'}. Best is trial#32 with value: 0.98.
[I 2020-07-07 08:37:25,838] Finished trial#46 with value: 0.98 with parameters: {'svc_c': 9.965028017522707, 'classifier': 'SVC'}. Best is trial#44 with value: 0.98.
[I 2020-07-07 08:37:26,126] Finished trial#50 with value: 0.96 with parameters: {'svc_c': 163518.71589804275, 'classifier': 'SVC'}. Best is trial#44 with value: 0.98.
[I 2020-07-07 08:37:26,528] Finished trial#53 with value: 0.94 with parameters: {'svc_c': 0.09024797035755694, 'classifier': 'SVC'}. Best is trial#44 with value: 0.98.
[I 2020-07-07 08:37:26,776] Finished trial#56 with value: 0.96 with parameters: {'svc_c': 6828.362148567885}. Best is trial#44 with value: 0.98.
[I 2020-07-07 08:37:27,114] Finished trial#59 with value: 0.9733333333333333 with parameters: {'svc_c': 11.879827968164516, 'classifier': 'SVC'}. Best is trial#44 with value: 0.98.
FrozenTrial(number=44, value=0.98, datetime_start=datetime.datetime(2020, 7, 7, 8, 37, 25, 403132), datetime_complete=datetime.datetime(2020, 7, 7, 8, 37, 25, 634957), params={'svc_c': 8.009547970836671, 'classifier': 'SVC'}, distributions={'svc_c': LogUniformDistribution(high=10000000000.0, low=1e-10), 'classifier': CategoricalDistribution(choices=('SVC', 'RandomForest'))}, user_attrs={}, system_attrs={}, intermediate_values={}, trial_id=45, state=TrialState.COMPLETE)

(venv) HideakinoMacBook-puro:simple mamu$ kubectl get pod
NAME           READY   STATUS      RESTARTS   AGE
postgres-0     1/1     Running     0          109s
worker-8wbpm   0/1     Completed   1          109s
worker-hxdf8   0/1     Completed   1          109s
worker-nvhw8   0/1     Completed   2          109s
worker-vjq2c   0/1     Completed   2          109s
worker-wkf47   0/1     Completed   1          109s
```

Therefore, I think it might be good to revert all changes on this file. What do you think?

Reply:
Sorry, for making you confused, but I did not mean to change anything of "simple" example except for the directory organization.